src/python/dxpy/scripts/dx.py

```diff
@@ -2522,26 +2522,49 @@ def wait(args):
 def build(args):
     sys.argv = ['dx build'] + sys.argv[2:]
 
+    def get_from_exec_desc(args):
+        """
+        Return source executable description when --from option is used
+        
+        """
+        exec_describe_fields={'fields':{"properties":True, "details":True},'defaultFields':True}
+        _, _, exec_result = try_call(resolve_existing_path,
```

Comment:
The resolved entity should be a single applet/workflow and the `exec_result` is a dict with field `id` and `description`.

Reply:
Could you add to the docstring what is expected in the `--from` argument (hash ID, qualified path, unqualified path, etc.)

Reply:
Added.